hardware_interface/src/fake_components/generic_system.cpp

```diff
@@ -54,6 +54,31 @@ return_type GenericSystem::configure(const hardware_interface::HardwareInfo & in
     }
   }
 
+  // Search for mimic joints
+  for (auto i = 0u; i < info_.joints.size(); ++i) {
+    const auto & joint = info_.joints.at(i);
+    if (joint.parameters.find("mimic") != joint.parameters.cend()) {
+      const auto mimicked_joint_it = std::find_if(
+        info_.joints.cbegin(), info_.joints.cend(),
+        [ & mimicked_joint = joint.parameters.at("mimic")](
+          const hardware_interface::ComponentInfo & joint_info) {
+          return joint_info.name == mimicked_joint;
+        });
+      if (mimicked_joint_it == info_.joints.cend()) {
+        throw std::runtime_error(
+                std::string(
+                  "Mimicked joint '") + joint.parameters.at("mimic") + "' not found");
+      }
+      MimicJoint mimic_joint;
+      mimic_joint.joint_index = i;
+      mimic_joint.mimicked_joint_index = std::distance(info_.joints.cbegin(), mimicked_joint_it);
+      auto param_it = joint.parameters.find("multiplier");
+      if (param_it != joint.parameters.end()) {
+        mimic_joint.multiplier = std::stod(joint.parameters.at("multiplier"));
```

Comment:
Does this default to `0` if it cant find the parameter? Wouldn't `1` be a better default?

Reply:
> Wouldn't 1 be a better default?

Yes, that's right, which's the default value see `double multiplier = 1.0;` in the `MimicJoint` struct definition